src/context.rs

```diff
@@ -803,3 +812,29 @@ mod private {
     impl IntoDecimalSealed for f64 {}
 }
 pub use private::{IntoBinary, IntoDecimal, IntoNumeral};
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+
+    macro_rules! check_expr {
+        ($ctx:expr, $expr:expr, $expected:expr) => {
+            let actual = $ctx.display($expr).to_string();
+            assert_eq!(actual, $expected);
+        };
+    }
+
+    #[test]
+    fn test_variadic_ops() {
+        let ctx = ContextBuilder::new().build().unwrap();
+
+        // As all variadic operators are generated by the `variadic!` macro above, we really only need
+        // to pick one to test.
+        check_expr!(ctx, ctx.imp_many([ctx.true_()]), "true");
+        check_expr!(
```

Comment:
I'm definitely open to changing this to the whole set of `*_many` ops. My feeling was that this made sense as a default behavior for all of them, but there might be a case I'm not thinking about.